firebase-firestore/src/test/java/com/google/firebase/firestore/remote/MockDatastore.java

```diff
@@ -26,6 +27,7 @@
 import com.google.firebase.firestore.model.mutation.MutationResult;
 import com.google.firebase.firestore.remote.WatchChange.WatchTargetChange;
 import com.google.firebase.firestore.spec.SpecTestCase;
+import com.google.firebase.firestore.testutil.EmptyAppCheckTokenProvider;
```

Comment:
I don't see this class in the PR, but I bet that we can combine EmptyAppCheckTokenProvider and EmptyCredentialsProvider.

Reply:
oops. it seems that the newly-added files weren't added to my commit, sorry. I've added them now.